forte/dmrg/block2_dmrg_solver.cc

```diff
@@ -192,7 +192,7 @@ struct Block2ScratchManager {
         // TODO may cause problem for MPI
         const std::vector<std::string> file_name_patterns = std::vector<std::string>{
             ".NPDM.FRAG.", ".MPS.",         ".MPS.INFO.", ".MMPS.", ".MMPS.INFO.",
-            ".MMPS-WFN.",  "-mps_info.bin", "@TMP.",      "@TMP-"};
+            ".MMPS-WFN.",  "-mps_info.bin", "@TMP.",      "@TMP-",  "DSRG-"};
```

Comment:
many of these intermediate scratch files are not documented - would appreciate it if you could  add some comment to explain a little bit what these files are here